wiki/Ranking_Criteria/Skinning/en.md

```diff
@@ -0,0 +1,339 @@
+Skin Set List 
```

Comment:
either, change this to say the folder's name or change the folder's name to match this